habitat/plan.sh

```diff
@@ -46,6 +46,14 @@ do_install() {
     gem install inspec-*.gem --no-document
   popd
 
+  # Binstub removed from gem distro, so it will not have been installed
+  # Do so manually
+  pushd "$pkg_prefix"
+    mkdir -p "$GEM_HOME/gems/inspec-${pkg_version}/bin/"
+    # Note that we generate a wrapper; this is the real executable
+    cp "$HAB_CACHE_SRC_PATH/$pkg_dirname/test/inspec-test-binstub" "$GEM_HOME/gems/inspec-${pkg_version}/bin/inspec"
```

Comment:
Something feels weird about our "real" binstubs living in the test folder, but I know this is the pattern we established so ᕕ( ᐛ )ᕗ